client/src/common/client.ts

```diff
@@ -38,6 +38,7 @@ import {
 	MessageStrategy, DidOpenTextDocumentParams, CodeLensResolveRequest, CompletionResolveRequest, CodeActionResolveRequest, InlayHintResolveRequest, DocumentLinkResolveRequest, WorkspaceSymbolResolveRequest
 } from 'vscode-languageserver-protocol';
 
+import { CancellationToken as JSONRPCCancellationToken } from 'vscode-jsonrpc';
```

Comment:
Protocol reexport vscode-jsonrpc. So you should get that type from `vscode-languageserver-protocol`

Reply:
Done.